client/actions/tests/agenda_test.js

```diff
@@ -349,10 +349,10 @@ describe('agenda', () => {
                     notify,
                     $timeout,
                 })
-                .then((planning) => {
-                    expect(planning).toEqual(plannings[0])
-                    // Cannot check dispatch(addPlanningToAgenda()) using a spy on dispatch
-                    // As addPlanningToAgenda is a thunk function
+                .then((plannings) => {
+                    expect(plannings).toEqual([plannings[0]])
```

Comment:
I think so. Nothing has changed on the planning. Only the agenda has been updated.